drivers/clock/sunxi-ccu.c

```diff
@@ -59,18 +53,32 @@ sunxi_ccu_enable(struct device *clockdev, struct device *dev)
 }
 
 static int
-sunxi_ccu_set_freq(struct device *clockdev __unused,
-                   struct device *dev __unused, uint32_t hz __unused)
+sunxi_ccu_get_rate(struct device *clockdev __unused,
+                   struct device *dev __unused, uint32_t *rate __unused)
+{
+	return ENOTSUP;
```

Comment:
Please consider adding a TODO comment here explaining the steps to set this up/why it cannot be setup at this time.